{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2020-9402",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "product_name": "n/a",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "n/a"
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    "vendor_name": "n/a"
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "Django 1.11 before 1.11.29, 2.2 before 2.2.11, and 3.0 before 3.0.4 allows SQL Injection if untrusted data is used as a tolerance parameter in GIS functions and aggregates on Oracle. By passing a suitably crafted tolerance to GIS functions and aggregates on Oracle, it was possible to break escaping and inject malicious SQL."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "n/a"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://docs.djangoproject.com/en/3.0/releases/security/",
                "refsource": "MISC",
                "name": "https://docs.djangoproject.com/en/3.0/releases/security/"
            },
            {
                "refsource": "MISC",
                "name": "https://groups.google.com/forum/#!topic/django-announce/fLUh_pOaKrY",
                "url": "https://groups.google.com/forum/#!topic/django-announce/fLUh_pOaKrY"
            },
            {
                "refsource": "CONFIRM",
                "name": "https://www.djangoproject.com/weblog/2020/mar/04/security-releases/",
                "url": "https://www.djangoproject.com/weblog/2020/mar/04/security-releases/"
            },
            {
                "refsource": "UBUNTU",
                "name": "USN-4296-1",
                "url": "https://usn.ubuntu.com/4296-1/"
            },
            {
                "refsource": "CONFIRM",
                "name": "https://security.netapp.com/advisory/ntap-20200327-0004/",
                "url": "https://security.netapp.com/advisory/ntap-20200327-0004/"
            }
        ]
    }
}